tests/integration/kafka/test_kafka_integration.py

```diff
@@ -1,8 +1,9 @@
 from datetime import datetime
 import os
 
-from kafka import KafkaAdminClient, KafkaProducer, KafkaConsumer
-from kafka.admin import NewTopic
+from kafka.admin import KafkaAdminClient, NewTopic
+from kafka.consumer import KafkaConsumer
+from kafka.producer import KafkaProducer
```

Comment:
Without this, mypy was raising errors:
```
tests/integration/kafka/test_kafka_integration.py:4: error: Module "kafka" has no attribute "KafkaAdminClient"  [attr-defined]
tests/integration/kafka/test_kafka_integration.py:4: error: Module "kafka" has no attribute "KafkaProducer"  [attr-defined]
tests/integration/kafka/test_kafka_integration.py:4: error: Module "kafka" has no attribute "KafkaConsumer"  [attr-defined]
```
As best I can tell, it was getting confused about the directory having the same name as the dependency. Changing the directory to something like `tests/integration/kafka_test/` seemed to work as well, so open to that if you prefer.

Reply:
done!